src/main/java/com/axibase/tsd/client/HttpClientManager.java

```diff
@@ -176,20 +178,20 @@ private HttpClient borrowClient() {
     }
 
     private void returnClient(HttpClient httpClient) {
-        objectPoolAtomicReference.get().returnObject(httpClient);
+        objectPoolAtomicReferenc.get().returnObject(httpClient);
     }
 
     private GenericObjectPool<HttpClient> createObjectPool() {
-        GenericObjectPool<HttpClient> httpClientGenericObjectPool = objectPoolAtomicReference.get();
+        GenericObjectPool<HttpClient> httpClientGenericObjectPool = objectPoolAtomicReferenc.get();
         if (httpClientGenericObjectPool == null) {
             httpClientGenericObjectPool = new GenericObjectPool<HttpClient>(new HttpClientBasePooledObjectFactory(), objectPoolConfig);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS2293)